### PR TITLE
(feat:) Added feature for expandable row component.  

### PR DESCRIPTION
**Pull request by java2kus**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Currently there is no feature for expandable row i.e. Detail row. #100 

**What is the new behavior?**
Provide an expandable detail row feature.

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by amcdnl** [comment]
THANK YOU! THIS IS AWESOME!

File: src/components/body/body.component.ts [hunk 1]
```diff
@@ -103,11 +108,28 @@ export class DataTableBody implements OnInit, OnDestroy {
...
+        // const offset = (this.state.options.rowHeight * action.limit) * action.offset;
+        this.scroller.setOffset(this.state.rowHeightsCache.query(rowIndex - 1));
       }
     });
 
+    this.sub.add(this.state.onExpandChange.subscribe( (expandedState) => {
+      // If there was more than one row expanded then there was a mass change
+      // in the data set hence adjust the scroll position.
+      if (expandedState.rows.length > 1) {
+        console.log(expandedState.currentIndex);
```
Comment (ocombe) [inline src/components/body/body.component.ts]: Is this a console.log that I see here ? :)
@amcdnl

Comment (java2kus) [inline src/components/body/body.component.ts]: @amcdnl @ocombe - my bad.  Not sure how it got missed :)   Thanks for pointing it out and pushing the fix.

Comment (ocombe) [inline src/components/body/body.component.ts]: it's ok, we already fixed it (since this has been merged) :)

**Pull request merged**